src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -240,11 +250,19 @@ bool pwmDshotMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t m
 #ifdef USE_DSHOT_TELEMETRY
     if (useDshotTelemetry) {
         output ^= TIMER_OUTPUT_INVERTED;
+        if (output & TIMER_OUTPUT_INVERTED) {
```

Comment:
Should this be conditional to STM32H7 as well?

Reply:
yes, as only the H7 has shown this issue currently.